base/docs/Docs.jl

```diff
@@ -12,7 +12,7 @@ const modules = Module[]
 
 const META′ = :__META__
 
-meta(mod) = mod.(META′)
+@eval meta(mod) = mod.$META′
```

Comment:
No not really, I have no clue what to look for. Dunno whether @ihnorton has been able to catch the freeze in gdb like I've done a handful of times. Or you could probably head in and do some work on the dual-boot laptop on @andreasnoack's desk during the week.